channels/microsoft365/index.rst

```diff
@@ -9,6 +9,7 @@ Microsoft 365
    signatures
    settings
    email-headers
+   M365errors
```

Comment:
The new chapter is part of a hidden table of contents, meaning it will show up only in sidebar and by using bottom navigation buttons.

Consider also adding a direct link to this chapter in form of a list item below. Take a look how `Header manipulation` was linked at the bottom of the document, and add one for `Common errors` and describe in a sentence what this chapter is about. Bonus points for identifying a suitable "emoji" to put in front of the list item 🛑